src/main/kotlin/com/msd/application/GameMapPlanetDto.kt

```diff
@@ -0,0 +1,19 @@
+package com.msd.application
+
+import com.msd.planet.domain.Planet
+import com.msd.planet.domain.PlanetType
+import lombok.NoArgsConstructor
+import java.util.*
+
+@NoArgsConstructor
```

Comment:
We have a compiler plugin for this. Why does it even need to have a no args constructor?